SECURITY.md

```diff
@@ -0,0 +1,23 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+If you think you found a vulnerability, and even if you are not sure about it, please send a message on
+[this discussion](https://github.com/encode/starlette/discussions/1861) sharing how we can contact you (e.g. your email).
+
+We'll contact you as soon as possible, with a proof that we are who we claim we are (you can also validate it 
+[here](https://github.com/orgs/encode/teams/operations)). Then you can reply with the vulnerability.
+
+Please try to be as explicit as possible, describing all the steps and example code to reproduce the security issue.
+
+We will review it thoroughly and get back to you.
```

Comment:
What do we think *is* the simplest way to have folks chat to us in a private(wish) way?

* A security email address?
* Contact one or more of us privately in our chat space? https://gitter.im/encode/community
* Comment on an issue with their contact details?
* Comment on an issue, requesting an invite, and then we temporarily add them to a security team and discuss on GitHub.

I'm thinking that the chat space might be an okay low-effort approach.
What do other teams in similar spaces to us do?

Reply:
Flask: https://github.com/pallets/flask/security/policy - security email address
Django: https://docs.djangoproject.com/en/dev/internals/security/ - security email address

👀

Reply:
We've already had some security issues in github discussions.. allows us to vet if the issue is serious or not, but for a serious exploit maybe PM a dev?